wallet/src/de/schildbach/wallet/BaseWalletApplication.kt

```diff
@@ -89,35 +73,9 @@ abstract class BaseWalletApplication : MultiDexApplication(), WalletDataProvider
         activity.startActivityForResult(sendCoinsIntent, requestCode)
     }
 
-    override fun sendCoins(address: Address, amount: Coin): LiveData<Resource<Transaction>> {
-        checkWalletCreated()
-        val wallet = walletApplication.wallet!!
-        val sendRequest = createSendRequest(address, amount)
-        val scryptIterationsTarget = walletApplication.scryptIterationsTarget()
-
-        return SendCoinsTask.sendCoins(wallet, sendRequest, scryptIterationsTarget)
-    }
-
-    override fun createSentDashAddress(address: String): Address {
-        return Address.fromString(Constants.NETWORK_PARAMETERS, address.toString().trim { it <= ' ' })
-    }
-
-    private fun createSendRequest(address: Address, amount: Coin): SendRequest {
-        return SendRequest.to(address, amount).apply {
-            coinSelector = ZeroConfCoinSelector.get()
-            useInstantSend = false
-            feePerKb = SendCoinsBaseViewModel.ECONOMIC_FEE
-            ensureMinRequiredFee = true
-        }
-    }
-
-    override fun getWalletBalance(): Coin {
```

Comment:
Balance is better to be observed than got once, but I left this method to not cause many changes. It's moved to the `WalletApplication`.